### PR TITLE
feat: Adds JWT authenticator implementation

### PR DESCRIPTION
**Pull request by hofmeister**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
+{
+    "name": "@kapeta/resource-type-rest",
```
Comment (sorenmat) [inline package.json]: is this correct?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    "version": "local",
+    "description": "Provides REST API and Client resource type to Kapeta",
```
Comment (sorenmat) [inline package.json]: what about this?

File: src/web/JWTAuthEditorComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+export const JWTAuthEditorComponent = (props: ResourceTypeProviderEditorProps) => {
+
+    return (
+        <>
+            <FormField
+                name={"metadata.name"}
+                label={"Name"}
+                validation={['required']}
+                help={"Name your site"}
```
Comment (sorenmat) [inline src/web/JWTAuthEditorComponent.tsx]: Seems a bit commanding :|

File: package.json (hunk 1 above)
Comment (hofmeister) [reply package.json]: Nope - fixed

File: package.json (hunk 2 above)
Comment (hofmeister) [reply package.json]: Also fixed

File: src/web/JWTAuthEditorComponent.tsx (hunk 3 above)
Comment (hofmeister) [reply src/web/JWTAuthEditorComponent.tsx]: Form was a copy past error - nothing to configure

**Pull request merged**